### DISCUSSION TITLE
wire:click fails parsing UUID

### DISCUSSION THREAD

**Discussion by Peter-Fogarty** [livewire/livewire#5228] [7. Bugs (archived)] [answered]
I have been trying to put a wire:click on a <TR> element where each row in my table is a standard laravel database notification.
Writing the quite boring code that looks like this,
```
@foreach ($notifications as $notification)
    <tr class="py-4 my-1 cursor-pointer bg-sky-200 hover:bg-green-300"
                      wire:key="row-{{ $loop->index }}" wire:click="toggleReadFlag({{ $notification->id }})">
           <td>{{ Arr::exists($notification->data, 'message') ? $notification->data['message'] : '' }}</td>
    </tr>
@endforeach
```
When I click on a tr in the webpage, it fails to execute my method, instead generating the following javascript error,
```
SyntaxError: No identifiers allowed directly after numeric literal
    function -- index.js:84
    value -- wire-directives.js:93
    get -- wire-directives.js:58
    (anonymous function) -- node_initializer.js:221
```

The error goes away completely and everything just works when I edit the database directly and replace the UUID strings with standard integers.

**Comment by gdebrauwer** [comment, marked as answer]
You have to add `'` around the id:
```html
<tr wire:click="toggleReadFlag('{{ $notification->id }}')"></tr>
```
